debian/rules

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS=hardening=+all
+
 %:
```

Comment:
No, I don't know how much performance impact it has. Can somebody can measuring that?